shared/build/script/docker.go

```diff
@@ -37,3 +44,31 @@ func DockerHostname(d *Docker) string {
 	}
 	return *d.Hostname
 }
+
+// DockerVolumes returns empty []string
+// when Docker.Volumes is empty.
+func DockerVolumes(d *Docker) []string {
```

Comment:
+1 on nil slice returning 0 on len function.  Wasn't thinking.  In running the unit tests i ran into an issue where b.Docker was nil, which is what made me make that method, just like the d.HostName method.  Originally I was just going to access it directly until unit tests started failing.